### ISSUE TITLE
v2.0.5 fails with "[TypeError: Cannot read property 'description' of null]"

### ISSUE THREAD

**Issue by astoilkov** [electron/windows-installer#41]
Running `electronInstaller.createWindowsInstaller` fails with `[TypeError: Cannot read property 'description' of null]`. I have tried to experiment by providing different options to the method but without success.

This is where the code fails in the compiled version. The `appMetadata` seems to have `null` value:
```
case 41:
  defaults = {
    // it fails here where appMetadata value is null
    description: appMetadata.description || '',
    exe: appMetadata.name + '.exe',
    iconUrl: 'https://raw.githubusercontent.com/atom/electron/master/atom/browser/resources/win/atom.ico',
    title: appMetadata.productName || appMetadata.name
  };
  metadata = _lodash2.default.defaults({}, options, defaults, appMetadata);


  if (!metadata.authors) {
    if (typeof metadata.author === 'string') {
      metadata.authors = metadata.author;
    } else {
      metadata.authors = (metadata.author || {}).name || '';
    }
  }

  metadata.owners = metadata.owners || metadata.authors;
  metadata.version = convertVersion(metadata.version);
  metadata.copyright = metadata.copyright || 'Copyright © ' + new Date().getFullYear() + ' ' + (metadata.authors || metadata.owners);

  _context3.t2 = _lodash2.default;
  _context3.next = 50;
  return _fsJetpack2.default.readAsync(_path2.default.join(__dirname, '..', 'template.nuspec'));
```

**Comment by valaxy** [comment]
same issue with the basic example code

**Comment by astoilkov** [comment]
That's probably because the published release and the current code are different. I think that the package is updated to the latest version in npm the issue will be fixed.

**Comment by havenchyk** [comment]
Guys, could you please try with just released 2.1.0?

**Comment by havenchyk** [comment]
I'm closing this. Feel free to reopen if you still have a problem.
